web/client/actions/catalog.js

```diff
@@ -66,12 +66,14 @@ export const SET_FORMAT_OPTIONS = 'CATALOG:SET_FORMAT_OPTIONS';
  * Adds a list of layers from the given catalogs to the map
  * @param {string[]} layers list with workspace to be added in the map
  * @param {string[] | object[] } sources catalog names related to each layer
+ * @param {string[]} options related to each layer. Can be used to overwrite default configuration or parameters
  */
-export function addLayersMapViewerUrl(layers = [], sources = []) {
+export function addLayersMapViewerUrl(layers = [], sources = [], options = []) {
```

Comment:
please amend or add the test for this action

Reply:
Updated existing test